website/docs/client/client-customization.mdx

```diff
@@ -3,6 +3,7 @@ id: client-customization
 title: Client Customization
 ---
 
+
```

Comment:
just curious why do we need 2 blank lines on top of docs?

Reply:
🤷🏻  That was added via the intellij "optimize imports"